salt/client/__init__.py

```diff
@@ -559,6 +559,7 @@ def cmd_batch(
             {'stewart': {...}}
         '''
         if 'expr_form' in kwargs:
+            import salt.utils.versions
```

Comment:
Can you add a comment here about why this later import is needed so we don't accidentally remove it in the future?